cli/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "whistle_cli"
+name = "whistle"
```

Comment:
revert back to whistle_cli

Reply:
add a [[[bin]]](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries) for the binary